packages/core/src/module.ts

```diff
@@ -15,7 +15,9 @@ export type Module = {
 
 type ModuleSpec = {
   properties: JSONSchema7;
-  events: string[];
+  events: {
```

Comment:
Here we'd better use `string[]` as `events` type, just like components.

Reply:
Here I am concerned about the need to store more content in the future, such as descriptions or otherwise. The `string[]` type will be a breaking change when it is extended in the future

Reply:
But this change itself is also a breaking change. And if events need more content, components spec also need change. Since component events spec is already `string[]` It will cause breaking change anyway.
Another reason is that If module events spec and component events spec are differenct, then the event code can not be reused between them.

Reply:
done